pages/common/fgrep.md

```diff
@@ -22,8 +22,8 @@
 
 - Display all lines except those that contain the given regular expression:
 
-`fgrep -v {{^regex$}} {{path/to/file}}`
+`fgrep -v {{^regular_expression$}} {{path/to/file}}`
 
 - Display filenames whose content matches the regular expression at least once:
 
-`fgrep -l {{^regex$}} {{path/to/file1}} {{path/to/file2}}`
+`fgrep -l {{^regular_expression}} {{path/to/file1}} {{path/to/file2}}`
```

Comment:
There seems to be a `$` missing here.

Reply:
Ah, thanks. Why do we even use `^...$` here?

Reply:
I thought the same, we are already calling it `regular_expression`, so no need to add `^...$`.

Reply:
Yeah. I saw different instances of this. Maybe it's a better idea to drop the `^` `$` completely :thinking: